default.cfg

```diff
@@ -1,16 +1,4 @@
 [starling]
-# Optional: If the SPS node is joined to One Identity Starling, you do not have to
```

Comment:
I'd mention here that "This plugin is only usable if the SPS node is joined to Starling."

Reply:
Done